datasets/cn/peoples_congress_wikipedia/crawler.py

```diff
@@ -139,3 +178,13 @@ def crawl(context: Context):
     for id, count in ids.items():
         if count > 1 and id not in IGNORE_DUPES:
             context.log.info(f"ID {id} emitted {count} times")
+    for h3 in doc.findall(".//h3"):
+        delegation_name = None
+        h3_text = h3.text_content().strip()
+        if any(heading in h3_text for heading in CHANGES_IN_REPRESENTATION):
+            table = h3.getparent().getnext()
+            if table.tag != "table":
+                table = table.getnext()
+            assert table.tag == "table"
+            for row in parse_table(context, table):
+                crawl_item(context, row, delegation_name)
```

Comment:
we can merge this with the block above by adding `any(heading in h3_text for heading in CHANGES_IN_REPRESENTATION)` to the `if not delegation_match:` then we only have the horrible code block once, right?